### DISCUSSION TITLE
Sort sessions by modification date in Telescope

### DISCUSSION THREAD

**Discussion by maxigaz** [olimorris/persisted.nvim#64] [Ideas]
It would be great if there was a way to sort sessions listed by the Telescope plugin by modification date, so that the session files in `~/.local/share/nvim/sessions` saved or updated more recently would appear at the bottom. This would be especially useful when autosave is enabled in persisted.nvim.

**Comment by olimorris** [comment]
Great idea and happy to accept a PR for it
